book/06-github/sections/2-contributing.asc

```diff
@@ -312,18 +312,18 @@ GitHub 風格的 Markdown 增加了許多你在基本 Markdown 語法做不到
 - [ ] 為程式碼做文件
 ----
 
-如果我們在 Pull Request 的描述裡或是議題裡加入這個，我們就能看到他被渲染成像 <<_task_lists>> 這樣。
+如果我們在 Pull Request 的描述裡或是議題裡加入這個，我們就能看到他被渲染成像 <<r_task_lists>> 這樣。
 
-[[_task_lists]]
+[[r_task_lists]]
 .Markdwon 評論裡渲染後的工作清單。
 image::images/markdown-02-tasks.png[工作清單範例]
 
 這個功能在 Pull Request 裏面，常被用來聲明在合併之前，你想要在這個分支裡完成的事情。最酷的地方就是你只要點下核取方塊就能更新你的評論－你不需要為了標記工作完成而得修改 Markdown。
 
-除此之外，GitHub 還會把議題和 Pull Request 裡面所有的工作清單整理起來，把它們作為後設資料顯示在 Pull Request 的清單頁面。舉例來說，如果你的 Pull Rewquest 裡面有工作清單，你可以在所有 Pull Request 的總覽頁面上看到進度。這讓人們得以把一個 Pull Request 分解成數個小工作，同時也便於其他人追蹤這個分支上的進度。你可以在 <<_task_list_progress>> 看到關於這個功能的範例。
+除此之外，GitHub 還會把議題和 Pull Request 裡面所有的工作清單整理起來，把它們作為後設資料顯示在 Pull Request 的清單頁面。舉例來說，如果你的 Pull Rewquest 裡面有工作清單，你可以在所有 Pull Request 的總覽頁面上看到進度。這讓人們得以把一個 Pull Request 分解成數個小工作，同時也便於其他人追蹤這個分支上的進度。你可以在 <<r_task_list_progress>> 看到關於這個功能的範例。
 
-[[_task_list_progress]]
-在 Pull Request 清單裡的工作清單統整。
+[[r_task_list_progress]]
+.在 Pull Request 清單裡的工作清單統整。
```

Comment:
should delete the dot?

Reply:
The title of the image was malformed. The dot is needed.